modules/vba_documents/app/controllers/vba_documents/internal/v1/upload_complete_controller.rb

```diff
@@ -68,16 +68,25 @@ def process_upload(upload_id)
           store = VBADocuments::ObjectStore.new
           raise Common::Exceptions::RecordNotFound, upload_id unless upload && store.bucket.object(upload.guid).exists?
 
+          log_details = { 'job' => self.class.name }.merge(upload.as_json)
+
           if upload.status.eql?('pending')
-            Rails.logger.info("VBADocuments: Processing: #{upload.inspect}")
-            upload.update(status: 'uploaded')
+            Rails.logger.info("VBADocuments: Started processing #{upload.class.name.demodulize} from S3", log_details)
```

Comment:
This log message was too similar to the "VBADocuments: Start Processing" log from the `VBADocuments::UploadProcessor`, so I modified it to mention S3 specifically. In addition, having `upload.inspect` in the message itself was causing spec failures (in CI only) due to a discrepancy in the `created_at`/`updated_at` timestamps in the expected vs. received response. Since we have the `log_details` passed as a separate argument now (which isn't failing in specs), we should be covered in terms of the `UploadSubmission` data still being available in the log.